package.json

```diff
@@ -37,6 +37,7 @@
     "cachedir": "^2.3.0",
     "chalk": "^2.4.2",
     "child-process-ext": "^2.1.1",
+    "ci-info": "^3.1.1",
```

Comment:
As I check the https://github.com/watson/ci-info/blob/master/CHANGELOG.md

We need to go with v2 in this case, otherwise we risk crashing for Node.js v6 users

Reply:
That is a great catch, you're totally right. Should be changed now